packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -37,9 +37,12 @@
             <input
                 v-else
                 :id="`${uniqueId}`"
+                :aria-labelledby="`label-${uniqueId}`"
                 :value="value"
                 v-bind="$attrs"
                 :type="normalisedInputType"
+                :min="normalisedInputType === 'number' ? minNumber : false"
```

Comment:
Worth checking, but I don't think you'll need the actual minNumber and maxNumber props.

The `v-bind="$attrs"` line just above will pass through any attributes assigned to the outer element directly onto the input – if you need to normalise those numbers, you could grab them off the `$attrs` passed in.

Reply:
So I think this could just be :min="normalisedInputType === 'number' ? $attrs['min'] : false"